qutebrowser/config/configdata.py

```diff
@@ -1239,6 +1239,8 @@ def data(readonly=False):
         ('stop', ['<Ctrl-s>']),
         ('print', ['<Ctrl-Alt-p>']),
         ('open qute:settings', ['Ss']),
+        ('select-follow', ['<Return>']),
```

Comment:
I think it actually makes sense to have a (hidden) command, because that's a fundamental and easy to understand concept as it is right now - a key is always mapped to a command, or passed through.

Let's leave it like this right now - maybe I should add some way to pass-through a key, but I think for now it's fine.